hlink/scripts/lib/conf_validations.py

```diff
@@ -279,10 +291,14 @@ def check_column_mappings(config, df_a, df_b):
                     raise ValueError(
                         f"Within a [[column_mappings]] the column_name: '{column_name}' does not exist in datasource_b and no previous [[column_mapping]] alias exists for it. Column mapping: {c}. Available columns: \n {df_b.columns}"
                     )
+        if column_name in columns_available or alias in columns_available:
+            duplicates.append(alias if alias else column_name)
```

Comment:
What about the situation where a user wants to read the same column in multiple times?

```
[[column_mappings]]
column_name = "namefrst"

[[column_mappings]]
column_name = "namefrst"
alias = "namefrst2"
```

This should be allowed.